model/model.js

```diff
@@ -144,3 +143,36 @@ RETURNING *;
         });
     });
 };
+
+exports.editVotes = ({ review_id }, voteInc) => {
+  if (isNaN(voteInc)) {
+    return Promise.reject({ status: 400, msg: `Invalid data type` });
+  }
+  return db
+    .query(
+      `
+  SELECT * FROM reviews
+  `
+    )
+    .then((allReviews) => {
+      if (review_id > allReviews.rows.length) {
+        return Promise.reject({ status: 400, msg: `Invalid id` });
+      }
```

Comment:
We can refactor this a bit to be more clear.

Do Select all from reviews where review_id = (the id being sent from the client)

you can then reject the promise if it returns no results (the review doesnt exist).